### ISSUE TITLE
Uncaught InvalidStateError: An attempt was made to use an object that is not, or is no longer, usable.

### ISSUE THREAD

**Issue by kasper21** [websanova/wPaint#48]
Hi,

I started to get this error and wpaint is not running because of it.

My ..wPaint() call is in a window load function.

Uncaught InvalidStateError: An attempt was made to use an object that is not, or is no longer, usable. wPaint.js:642
Canvas.getImage wPaint.js:642
Canvas.addUndo wPaint.js:690
(anonymous function) wPaint.js:128
e.extend.each jquery.js?ver=1.7.2:2
e.fn.e.each jquery.js?ver=1.7.2:2
$.fn.wPaint wPaint.js:58
(anonymous function) (index):332
f.event.dispatch jquery.js?ver=1.7.2:3
h.handle.i

any help would be appreciated!

**Comment by websanova** [comment]
Do you have all the libraries included for jquery ui?

**Comment by kasper21** [comment]
yeah I do. I fixed it though. was trying to use resizable canvas but hadn´t updated to 2.0 version.

now I get a different problem with vers 2.x: 

Uncaught TypeError: Object bg has no method 'capitalize' wPaint.js:35
createCanvas wPaint.js:35
Paint.generate wPaint.js:82
Paint wPaint.js:20
get wPaint.js:1102
elCreateEach wPaint.js:1091
e.extend.each jquery.js?ver=1.7.2:2
e.fn.e.each jquery.js?ver=1.7.2:2
$.fn.wPaint wPaint.js:1124
(anonymous function) (index):370
f.event.dispatch jquery.js?ver=1.7.2:3
h.handle.i jquery.js?ver=1.7.2:3

**Comment by kasper21** [comment]
I didn´t realize the new version has plugin files. I linked them now. Like this:
    <script type="text/javascript" src="http://rvw.bespokestudio.com/wp-content/themes/proc3ss/js/wPaint.js"></script>

<script type="text/javascript" src="http://rvw.bespokestudio.com/wp-content/themes/proc3ss/js/wPaint.menu.main.min.js"></script>

<script type="text/javascript" src="http://rvw.bespokestudio.com/wp-content/themes/proc3ss/js/wPaint.menu.text.min.js"></script>

<script type="text/javascript" src="http://rvw.bespokestudio.com/wp-content/themes/proc3ss/js/wPaint.menu.main.shapes.min.js"></script>

<script type="text/javascript" src="http://rvw.bespokestudio.com/wp-content/themes/proc3ss/js//wPaint.menu.main.file.min.js"></script>

but now besides this error:
Object bg has no method 'capitalize' wPaint.js:35

I get this error as well:
TypeError: 'undefined' is not a function (evaluating '$("#wPaint").wPaint("resize")')

**Comment by websanova** [comment]
Can you give short sample of your code.

**Comment by kasper21** [comment]
hey websanova,

no worries. I got it. now a question comes up. I used the resize function and I learned that the canvas does resize
but I looses the currently drawn data. Is there a way of re-initalizing the plugin after window resize?
I tried storing the base64 value in a variable, empty the main #wPaint and call the wPaint() function again with the
stored base64 string as-  image: value, but I get an empty canvas.

any help would be greatly appreciated.

thanks,
Kasper

**Comment by websanova** [comment]
Yes, just call $('#wPaint').wPaint('resize'); after you resize the parent
#wPaint element.

**Comment by kasper21** [comment]
The thing is, that I´ve been using an older version of the plugin, but resize wasn´t a function in that version.
Now, when I update to the latest version I get this error again:

Uncaught TypeError: Object bg has no method 'capitalize' wPaint.js:35

However, I tried the fullscreen.html example and I figured that resizing does resize the canvas but doesn´t redraw proportionally. So this won´t be a solution anyways.

Thanks!
